WireBackup/Sources/WireBackup/KaliumBackupAdapters/BackupCreator/BackupCreator+addMessage.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+@preconcurrency import KaliumBackup
+
+extension BackupCreator {
+
+    func addMessage(_ message: some MessageEntityProtocol) {
+
+        let backupMessage = BackupMessage(
+            id: message.id,
+            conversationId: BackupQualifiedId(message.conversationID),
+            senderUserId: BackupQualifiedId(message.senderUserID),
+            senderClientId: message.senderClientID ?? "",
+            creationDate: BackupDateTime(message.creationDate),
+            content: BackupMessageContent(message.content),
+            webPrimaryKey: nil
+        )
+        mpBackupCreator.add(message: backupMessage)
+
+    }
+
+    private func BackupMessageContent(_ content: MessageContent) -> KaliumBackup.BackupMessageContent {
```

Comment:
suggestion/question: I think it can be a bit confusing to hide the fact that is this a regular method. Is there a particular reason for it? Same with the other similar methods

```suggestion
    private func backupMessageContent(_ content: MessageContent) -> KaliumBackup.BackupMessageContent {
```

Reply:
done